resources/packages/ShotgunBridge/configure.js

```diff
@@ -1047,6 +1047,53 @@ function Engine()
         return scene.isDirty();
     }
 
+    self.render_image_sequence = function (data)
+    {
+        MessageLog.trace("RENDER TO IMAGE SEQUENCE");
+
+        //Make sure the destination dir exists if not create it
+        var scenePath = scene.currentProjectPath();
+        var framesPath = "C:/Users/RT/Desktop/frames";
```

Comment:
Can javascript do path expansion?  This might be better expressed as "~/Desktop/frames" to support our eventual non-RT-user-future.